src/screens/ENSAssignRecordsSheet.js

```diff
@@ -81,12 +81,13 @@ export default function ENSAssignRecordsSheet() {
   });
 
   const [avatarUrl, setAvatarUrl] = useState(initialAvatarUrl);
-
   const [accentColor, setAccentColor] = useRecoilState(accentColorAtom);
+
   const { result: dominantColor } = usePersistentDominantColorFromImage(
-    avatarUrl || ''
+    avatarUrl
```

Comment:
If `avatarUrl` is empty, then this would produce a TS error?

Reply:
added again, is a js file so it didn't say anything